Purchases/Purchasing/Purchases.swift

```diff
@@ -876,9 +876,27 @@ public extension Purchases {
     ///   ...
     /// }
     /// ```
-    /// - Note: an alternative way of getting ``CustomerInfo`` updates
-    /// is using ``PurchasesDelegate/purchases(_:receivedUpdated:)``
-    /// - Note: this method is not thread-safe.
+    /// #### SwiftUI Example:
+    /// ```swift
+    /// ContentView()
+    ///   .environmentObject(revenueCatCustomerData)
+    ///   .task {
+    ///     if let customerInfo = try? await Purchases.shared.customerInfo() {
```

Comment:
This isn't needed now that the stream starts with the first value (https://github.com/RevenueCat/purchases-ios/pull/1257)

Reply:
nice